recipes/d3ploy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "d3ploy" %}
+{% set version = "0.8" %}
+{% set sha256 = "f98c4984974a834963b474be394f10aac718d72ba5ea48c0da2ca098ff202943" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ergs/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
from the example, I think the new standard is to use:

`python -m pip install --no-deps --ignore-installed .`